public class TodoFixMeCatchBlock {

    /**
     * A test function that overcatches Exceptions and performs a System.exit
     */
    public static void testException() {
        try {

        } catch (Exception e) {
            // TODO: a todo comment
            return;
        }
    }

}